documentation/source/building.rst

```diff
@@ -292,6 +292,18 @@ The following variables are makefile variables, not environment variables.
       to set the random seed value if :envvar:`RANDOM_SEED` is not set.
       If both ``+ntb_random_seed`` and ``+seed`` are set, ``+ntb_random_seed`` is used.
 
+.. make:var:: CMD_PREFIX
+
+      Prefix added to invocations of $(CMD)
+
+      Can be used to add environment variables or prefixed commands to invocations of
+      $(CMD). This feature has been motivated by `issue 644 <https://github.com/cocotb/cocotb/issues/644>`_
+      to work work around issues with conflicting dynamic libraries. In these situations
+      users can set ``CMD_PREFIX := LD_PRELOAD="foo.so bar.so"`` to force a particular
+      library.
+
+      .. versionadded:: 1.5.3
```

Comment:
I've set it to `1.5.3`, but we might want to wait on a major release?

Reply:
```suggestion
      .. versionadded:: 1.6.0
```

The master branch is for the next feature release. If this were to go into a patch release this commit would need to be cherry-picked into the `stable/1.5` branch. Since this is a new non-breaking feature and not a bugfix, it should go in the next feature release.